docs/pages/index.rst

```diff
@@ -9,26 +9,74 @@ When rendering a scene from the Matterport3D dataset, *Habitat-Sim* achieves
+=================================================== ======================================================================================== ======================
+Basics for Navigation                               `Video <https://youtu.be/kunFMRJAu2U?list=PLGywud_-HlCORC0c4uj97oppQrGiB6JNy>`__                                     `Interactive Colab <https://colab.research.google.com/github/facebookresearch/habitat-sim/blob/master/examples/tutorials/colabs/ECCV_2020_Navigation.ipynb>`__
 
-.. _ECCV 2020 tutorial series: https://aihabitat.org/tutorial/2020/
+Interaction                                         `Video <https://youtu.be/6eh0PBesIgw?list=PLGywud_-HlCORC0c4uj97oppQrGiB6JNy>`__                                     `Interactive Colab <https://colab.research.google.com/github/facebookresearch/habitat-sim/blob/master/examples/tutorials/colabs/ECCV_2020_Interactivity.ipynb>`__
 
-.. block-info:: C++ API documentation
+Advanced Topics                                     `Video <https://youtu.be/w_kDq6UOKos?list=PLGywud_-HlCORC0c4uj97oppQrGiB6JNy>`__                                     `Interactive Colab <https://colab.research.google.com/github/facebookresearch/habitat-sim/blob/master/examples/tutorials/colabs/ECCV_2020_Advanced_Features.ipynb>`__
 
-    Habitat-sim is designed to be used primarily through its Python API. As such, the
-    end-user tutorials and docs linked above focus on Python.
+Profiling and Optimization                          `Video <https://youtu.be/I4MjX598ZYs?list=PLGywud_-HlCORC0c4uj97oppQrGiB6JNy>`__                                     `Interactive Colab <https://colab.research.google.com/gist/eundersander/b62bb497519b44cf4ceb10e2079525dc/faster-rl-training-profiling-and-optimization.ipynb>`__
 
-    If you're looking for API reference of Habitat-sim's C++ internals, please see the
-    `C++ API <cpp.html>`_ tab.
+New Actions                                         :ref:`Web page <std:doc:new-actions>`
+
+Notebooks                                           :ref:`Web page <std:doc:notebooks>`
+
+Creating a stereo agent                             :ref:`Web page <std:doc:stereo-agent>`
+
+Working with light setups                           :ref:`Web page <std:doc:lighting-setups>`
+
+Extracting Images                                   :ref:`Web page <std:doc:image-extractor>`
+
+Interactive Rigid Objects                           :ref:`Web page <std:doc:rigid-object-tutorial>`
+=================================================== ======================================================================================== ======================
+
+Python Classes
+==============
+
+See the `Classes <./classes.html>`_ tab.
+
+Python Unit Tests
+=================
+
+Browse selected unit tests that demonstrate essential *Habitat-Sim* interfaces.
+
+- `test_agent.py <https://github.com/facebookresearch/habitat-sim/blob/master/tests/test_agent.py>`_
```

Comment:
We should make an RST macro for the repo if we don't already. These are quite a lot of long URLS.

Reply:
Sadly, RST substitutions don't work inside link URLs. I couldn't find any RST or m.css solution here. @mosra any ideas?

It's not much help, but I'll switch to the m.css `:gh:` macro to least avoid avoid typing `https://www.github.com/`.

Reply:
We have one for :gh: but I don't think it worked for individual files.